backend/apps/admin/src/problem/problem.service.ts

```diff
@@ -0,0 +1,104 @@
+import { Injectable } from '@nestjs/common'
+import { isDefined } from 'class-validator'
+import { PrismaService } from '@libs/prisma'
+import type { ProblemWhereInput } from '@admin/@generated/problem/problem-where.input'
+import type { CreateGroupProblemDto } from './dto/create-problem.dto'
+import type {
+  DeleteGroupProblemDto,
+  GetGroupProblemDto,
+  GetGroupProblemsDto
+} from './dto/request-problem.dto'
+import type { UpdateProblemDto } from './dto/update-problem.dto'
+
+@Injectable()
+export class ProblemService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async create(createdById: number, problemCreateInput: CreateGroupProblemDto) {
+    const problemTestcase = {
+      create: problemCreateInput.testcase
+    }
+
+    let tagsId
+    if (isDefined(problemCreateInput.tag)) {
+      tagsId = problemCreateInput.tag.map((value) => {
+        return {
+          tagId: value
+        }
+      })
+    }
+
+    const { testcase, tag, ..._data } = problemCreateInput
+
+    return await this.prisma.problem.create({
+      data: {
+        ..._data,
+        createdById: createdById,
+        problemTestcase: problemTestcase,
```

Comment:
```suggestion
        problemTestcase: {
          create: testcases
        },
```
여기도 위의 로직을 이렇게 바꿔주는 게 더 깔끔할 것 같긴 한데, 이 부분은 나중에 수정될 부분이라 크게 신경쓰지 않으셔도 될 것 같습니다.